backend/src/LibExecution/ProgramTypes.fs

```diff
@@ -505,6 +576,26 @@ module UserType =
       deprecated : Deprecation<TypeName.T> }
 
 
+
+
+type Const =
+  | CInt of int64
+  | CBool of bool
+  | CString of string
+  | CChar of string
+  | CFloat of Sign * string * string
+  | CUnit
```

Comment:
I wonder - when would I use a Const of a Unit?

Reply:
Perhaps

```
let myError = Error ....
let mySuccess = Ok ()
```

Reply:
Good point - thanks!